code/modules/awaymissions/pamphlet.dm

```diff
@@ -30,8 +30,8 @@
 			number of cases, the Gateway they have established may not be immediately obvious. \
 			Do not panic if you cannot locate the return Gateway. Begin colonisation of the destination. \
 			<br><br><b>A New World</b><br>\
-			As a participant in the Nanotrasen Gateway Project, you will be on the frontiers of space. \
-			Though complete safety is assured, participants are advised to prepare for inhospitable \
+			As a participant in the Horizons Gateway Project, you will be on the frontiers of space. \
+			Though we've prepared you for a safe journey, participants are advised to prepare for inhospitable \
 			environs."

```

Comment:
Should this be `environments`? I dont really know

Reply:
Yes. Yes it should.
Environs basically means encircling.